hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AuthenticationBaseMessageTask.java

```diff
@@ -172,25 +172,19 @@ private ClientMessage prepareUnauthenticatedClientMessage() {
         logger.warning("Received auth from " + connection + " with clientUuid " + clientUuid
                 + " and clientName " + clientName + ", authentication failed");
         byte status = CREDENTIALS_FAILED.getId();
-        return encodeAuth(status, null, null, serializationService.getVersion(),
-                clientEngine.getPartitionService().getPartitionCount(), clientEngine.getClusterService().getClusterId(),
-                clientFailoverSupported);
+        return encodeAuth(status, null, null, serializationService.getVersion(), -1, null, clientFailoverSupported, false);
```

Comment:
Do we need to return the `serializationService.getVersion()`? The un-authenticated client shouldn't need it in my opinion

Reply:
Ok removing it as well.